extensions/amp-selector/0.1/amp-selector.js

```diff
@@ -324,38 +324,32 @@ export class AmpSelector extends AMP.BaseElement {
     if (el.hasAttribute('disabled')) {
       return;
     }
-
     this.mutateElement(() => {
-      /** @type {?Array<string>} */
-      let selectedValues;
       if (el.hasAttribute('selected')) {
         if (this.isMultiple_) {
           this.clearSelection_(el);
-          selectedValues = this.setInputs_();
+          this.setInputs_();
         }
       } else {
         this.setSelection_(el);
-        selectedValues = this.setInputs_();
-      }
-
-      // Don't trigger action or update focus if
-      // selected values haven't changed.
-      if (selectedValues) {
```

Comment:
`selectedValues` is always truthy since `setInputs_()` returns `!Array`. This was just wrong.